index.js

```diff
@@ -0,0 +1,82 @@
+// ## Question One
+
+// Write a function that takes in a callback and calls the callback
+
+// const newFunction = (callback) => {return callback}
+
+// console.log(newFunction("Hello"))
+
+// Question two
+
+// const secondFunc = (callback, boolean) => {
+//     if(boolean === true ){
+//         return callback
```

Comment:
need to invoke cb

Reply:
This appears to be a pattern throughout. You MUST test the code you write to make sure that it is working properly.